Doc/library/functions.rst

```diff
@@ -892,6 +892,10 @@ are always available.  They are listed here in alphabetical order.
    sequence (such as a string, bytes, tuple, list, or range) or a collection
    (such as a dictionary, set, or frozen set).
 
+   **CPython implementation detail:**  ``len`` raises :exc:`OverflowError`
```

Comment:
There is a directive for implementation details:
```
.. impl-detail::
```

Reply:
Thanks @csabella - done :smile: